app/screens/create_direct_message/create_direct_message.tsx

```diff
@@ -239,7 +145,7 @@ export default function CreateDirectMessage({
 
             clearSearch();
         }
-    }, [selectedIds, currentUserId, handleRemoveProfile, startConversation, clearSearch]);
+    }, [clearSearch, currentUserId, handleRemoveProfile, onButtonTap, selectedIds, setSelectedIds]);
```

Comment:
`setSelectedIds` comes from a state set function, so there is no need to have it in the dependencies.